environment/lib/eth-client-setup/index.js

```diff
@@ -2,145 +2,171 @@ const nearlib = require('nearlib');
-            deps: {
-                keyStore: this.keyStore,
-            },
-        });
-
-        // const masterAccount = new nearlib.Account(this.near.connection, this.masterAccountId);
-        // const balance = new BN('1000000000000000000000000000');
-        // await masterAccount.createAccount('foobarbar6', this.ethClientPK, balance);
-
-        // Initialize accounts and deploy the contracts. Call initialization functions if needed.
-        await this.verifyAccount(this.masterAccountId);
-        // await this.maybeCreateAccount('foobarbar7', this.ethClientPK, this.ethClientInitBalance, this.ethClientContractPath);
-
-        await this.maybeCreateAccount(this.ethClientAccId, this.ethClientPK, this.ethClientInitBalance, this.ethClientContractPath);
-        await this.verifyAccount(this.ethClientAccId);
-        this.ethClientAccount = new nearlib.Account(this.near.connection, this.ethClientAccId);
-        this.ethClientContract = new EthClientContract(this.ethClientAccount, this.ethClientAccId);
-        await this.ethClientContract.maybeInitialize(this.validateEthash == 'true');
-
-        await this.maybeCreateAccount(this.ethProverAccId, this.ethProverPK, this.ethProverInitBalance, this.ethProverContractPath);
-        this.ethProverAccount = new nearlib.Account(this.near.connection, this.ethProverAccId);
-        this.ethProverContract = new EthProverContract(this.ethProverAccount, this.ethProverAccId);
-        await this.ethProverContract.maybeInitialize(this.ethClientAccId);
+  constructor() {}
+  async initialize() {
+    this.nearNodeURL = process.env.NEAR_NODE_URL;
+    this.nearNodeNetworkId = process.env.NEAR_NODE_NETWORK_ID;
+
+    this.masterAccountId = process.env.MASTER_ACC_ID;
+    this.masterAccountSK = process.env.MASTER_SK;
+
+    this.ethClientAccId = process.env.ETH_CLIENT_ACC_ID;
+    this.ethClientSK = process.env.ETH_CLIENT_SK;
+    this.ethClientPK = nearlib.KeyPair.fromString(this.ethClientSK).publicKey;
+    this.ethClientInitBalance = process.env.ETH_CLIENT_INIT_BALANCE;
+    this.ethClientContractPath = process.env.ETH_CLIENT_CONTRACT_PATH;
+
+    this.ethProverAccId = process.env.ETH_PROVER_ACC_ID;
+    this.ethProverSK = process.env.ETH_PROVER_SK;
+    this.ethProverPK = nearlib.KeyPair.fromString(this.ethProverSK).publicKey;
+    this.ethProverInitBalance = process.env.ETH_PROVER_INIT_BALANCE;
+    this.ethProverContractPath = process.env.ETH_PROVER_CONTRACT_PATH;
+    this.validateEthash = process.env.VALIDATE_ETHASH;
+
+    this.nearTokenAccId = process.env.NEAR_TOKEN_ACC_ID;
+    this.nearTokenSK = process.env.NEAR_TOKEN_SK;
+    this.nearTokenPK = nearlib.KeyPair.fromString(this.nearTokenSK).publicKey;
+    this.nearTokenInitNearBalance = process.env.NEAR_TOKEN_INIT_NEAR_BALANCE;
+    this.nearTokenContractPath = process.env.NEAR_TOKEN_CONTRACT_PATH;
+
+    this.nearLockerAccId = process.env.NEAR_LOCKER_ACC_ID;
+    this.nearLockerSK = process.env.NEAR_LOCKER_SK;
```

Comment:
we have overwhelm number of env vars, make it not convenient for local and for deployment. Next step we could add dot-env, and push a .env.example

Reply:
nvm, @chefsale provide these from start command which looks good, we don't need dotenv